src/main/webapp/scripts/private-room-script.js

```diff
@@ -1,16 +1,25 @@
+const STATE_LISTENER_TIMER_MS = 1000;
+const PLAYER_SECONDS_DISCREPANCY = 2;
```

Comment:
this is fine to me as a quick solution so far. In actual project, you should use enum defined by YT API and get index by the enum value list instead of hard coding them.